src/js/synth.js

```diff
@@ -61,11 +81,11 @@ document.addEventListener("keydown", function (event) {
   }
 
   // bail, if a modifier is pressed alongside the key
-  if (event.ctrlKey || event.shiftKey || event.altKey || event.metaKey) {
+  if (event.ctrlKey || event.shiftKey || event.altKey || event.metaKey || event.repeat) {
```

Comment:
I'm gonna add the change from this line to the repo and possibly refactor this line into a function